### ISSUE TITLE
AmberParmBuilder test case assertion error

### ISSUE THREAD

**Issue by graik** [graik/biskit#20]
Tested using AmberTools 17 on Ubuntu 16.04

Something wrong with chain break detection:
```
FAIL: test_AmberParmSolvated (__main__.Test)
AmberParmBuilder.parmSolvated test
----------------------------------------------------------------------
Traceback (most recent call last):
  File "AmberParmBuilder.py", line 669, in test_AmberParmSolvated
    self.assertEqual( self.ref.lenChains(), self.m3.lenChains() )
AssertionError: 3 != 2

----------------------------------------------------------------------
```

**Comment by graik** [comment]
fixed in `biskit3` branch but not yet in master

**Comment by graik** [comment]
closing -- this bug-fix will never be back-ported to the 2.x version but is available from 3.0
